src/components/pages/Home/UseCasesSection/index.tsx

```diff
@@ -297,24 +358,55 @@ const UseCasesSection: React.ForwardRefRenderFunction<HTMLElement> = () => (
                     <ExampleBox title="CML Report">
-                      <a target="_blank" rel="noreferrer" href="https://github.com/iterative/cml_tensorboard_case/pull/1">
+                      <a target="_blank" rel="noreferrer" href="https://github.com/iterative-test/cml-example-tensorboard/pull/1">
                         <Image src="/img/github/tensorboard-report.png" alt="GitHub Tensorboard report example" />
                       </a>
                     </ExampleBox>
-
+        
+                  </Collapser>
+                )}
+                bitbucket={(
+                  <Collapser>
+                    <Code filename="bitbucket-pipelines.yml" repo="https://bitbucket.org/iterative-test/cml-example-tensorboard">
+                      <div><span>image: iterativeai/cml:0-dvc2-base1</span></div>
+                      <div><span>pipelines:</span></div>
+                      <div><span>  default:</span></div>
+                      <div><span>    - step:</span></div>
+                      <div><span>        name: Train model</span></div>
+                      <div><span>        script: </span></div>
+                      <Tooltip type="tensorboard">
+                        <div><span>          - pip install -r requirements.txt</span></div>
+                        <div><span>          - cml tensorboard connect</span></div>
+                        <div><span>            --logdir=./logs</span></div>
+                        <div><span>            --name=&quot;Go to tensorboard&quot;</span></div>
+                        <div><span>            --md &gt;&gt; report.md</span></div>
+                      </Tooltip>
+                      <Tooltip type="reports">
+                        <div><span>          - cml comment create report.md</span></div>
+                      </Tooltip>
+                      <div><span>      </span></div>
+                      <Tooltip type="dependencies">
+                        <div><span>          - python train.py  # generate ./logs</span></div>
+                      </Tooltip>
+                    </Code>
+                    <ExampleBox title="CML Report">
+                      <a target="_blank" rel="noreferrer" href="https://bitbucket.org/iterative-test/cml-example-tensorboard">
+                        <Image src="/img/bitbucket/tensorboard-report.png" alt="Bitbucket Tensorboard report example" />
+                      </a>
+                    </ExampleBox>
+        
                   </Collapser>
                 )}
               />
             )
           },
-          ,
           {
             name: "CML Runner Cloud",
             content: (
               <Switchable
                 gitlab={(
                   <Collapser>
-                    <Code filename=".gitlab-ci.yml" repo="https://gitlab.com/iterative.ai/cml-runner-example">
+                    <Code filename=".gitlab-ci.yml" repo="https://gitlab.com/iterative-test/cml-example-cloud-gpu">
```

Comment:
Also this is a non-GPU example, vis. https://github.com/iterative/cml-examples-dev/pull/22

```suggestion
                    <Code filename=".gitlab-ci.yml" repo="https://gitlab.com/iterative.ai/cml-runner-example">
```